core/src/com/unciv/logic/map/MapGenerator.kt

```diff
@@ -653,6 +665,38 @@ class MapGenerator(val ruleset: Ruleset) {
             return Perlin.noise3d(worldCoords.x.toDouble(), worldCoords.y.toDouble(), seed, nOctaves, persistence, lacunarity, scale)
         }
 
+        private fun getDistortedPerlinNoise(tile: TileInfo, seed: Double,
+                                   nOctaves: Int = 6,
+                                   persistence: Double = 0.5,
+                                   lacunarity: Double = 2.05,
+                                   scale: Double = 20.0): Double {
+            val worldCoords = HexMath.hex2WorldCoords(tile.position)
+
+            var noise = Perlin.noise3d(worldCoords.x.toDouble()+100.5, worldCoords.y.toDouble()+100.5, seed+100.5, 1, persistence, lacunarity, scale*20.0)
+            var lacunarityNoise = (noise*1.0+1.0)*lacunarity*0.7
+
+            noise = Perlin.noise3d(worldCoords.x.toDouble(), worldCoords.y.toDouble(), seed-1000.5, 1, persistence, lacunarity, scale*20.0)
+            var persistenceNoise = (noise*1.0+1.0)*persistence
+            persistenceNoise = min(persistenceNoise, 0.5)
+
+            noise = Perlin.noise3d(worldCoords.x.toDouble(), worldCoords.y.toDouble(), seed+733.1, 1, persistence, lacunarity, scale*20.0)
+            var distortion_x = noise*150.0
+
+            noise = Perlin.noise3d(worldCoords.x.toDouble(), worldCoords.y.toDouble(), seed-334.4, 1, persistence, lacunarity, scale*20.0)
+            var distortion_y = noise*150.0
+
+            return Perlin.noise3d(
+                    worldCoords.x.toDouble() + distortion_x,
+                    worldCoords.y.toDouble() + distortion_y,
+                    seed,
+                    nOctaves,
+                    persistenceNoise,
+                    lacunarityNoise,
+                    scale,
+                    1.0)
+        }
+
+
```

Comment:
A perlin noise which uses multiple perlin noise to change its parameters. Lacunarity, persistence, and x and y displacements are modified using perlin noise